src/Models/ShallowWaterModels/calculate_shallow_water_tendencies.jl

```diff
@@ -77,19 +83,19 @@ function calculate_interior_tendency_contributions!(tendencies,
 
     barrier = Event(device(arch))
 
-    Guh_event = calculate_Guh_kernel!(tendencies.uh,
+    Guh_event = calculate_Guh_kernel!(tendencies[1],
```

Comment:
Given that the only difference between the next 3 steps is the index, would it be more compact to put these in a for loop over `1,2,3`?

Reply:
I like it written out personally!

Reply:
I am happy either way.  Just a thought.